pages/nav.css

```diff
@@ -51,9 +51,10 @@ nav {
   gap: 2px;
   grid-area: nav;
   margin: 0 3rem;
-  padding-top: 4rem;
+  padding: 4rem 0;
   font-size: 1.6rem;
   position: relative;
+  overflow: scroll;
```

Comment:
make overflow: auto so it only appear when needed.

Reply:
I have hidden the scrollbar